lib/src/reactive_ble.dart

```diff
@@ -98,14 +103,21 @@ class FlutterReactiveBle {
   Future<void> _initialization;
 
   ScanSession _currentScan;
+  DeviceConnector _deviceConnector;
 
   /// Initializes this [FlutterReactiveBle] instance and its platform-specific
   /// counterparts.
   ///
   /// The initialization is performed automatically the first time any BLE
   /// operation is triggered.
   Future<void> initialize() async {
+    _pluginController ??=
+        PluginController(ArgsToProtobufConverter(), _methodChannel);
     _initialization ??= _methodChannel.invokeMethod("initialize");
+    _deviceConnector ??= DeviceConnector(
+      connectionStateUpdateStream: connectedDeviceStream,
+      pluginController: _pluginController,
+    );
     _prescanConnector = PrescanConnector(
```

Comment:
This suspicious line... (just so that we remember)

Reply:
it is very suspicious. Next pr will to move the prescan connector inside the deviceconnector. I have a nice idea how to fix this